source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+# Tutorial: Sensor Configuration, Feedback, & Profiling
+
+- **Goal:** Learn how to use the AMDC Timing Manager to get feedback from sensors
+- **Complexity:** 4 / 5
+- **Estimated Time:** TODO
+
+
+## Tutorial Requirements
+
+1. Working AMDC Hardware
+2. Completion of the [Hardware Commands](/getting-started/tutorials/hw-commands/index.md) tutorial
+3. Completion of the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial
+4. Completion of the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial
+
+This tutorial expands on the code created in the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial, and uses commands created in the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial. Both must be completed before this tutorial.
+
+## Introduction 
+
+In motor control applications, it is important to have consistent timing of both sensor data acquisition and the running of control tasks. In default configuration, the AMDC allows for some jitter between sensor measurement time and control loop timing. This tutorial shows how the AMDC can be configured to elimiate this jitter.
+
+This tutorial provides the source code for:
+* experimenting the effects of the timing manager using the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) app
+
+## Scheduling and Synchronizing:
+
+The AMDC synchronizes running tasks and sensor collection to the PWM carrier wave. Every X PWM periods (where X is set by the function timing_manager_set_ratio()), the AMDC will collect data from sensors. In Legacy mode, the AMDC will run control tasks concurrently with sensor collection. In Post-sensor mode the AMDC will not run control tasks until the sensor collection is complete. 
+
+There are multiple factors that affect when and how fast control tasks run.
+ - User set TASK_UPDATES_PER_SEC
+ - PWM Frequency
+ - User Event Ratio
+ - Sensor collection time (post-sensor mode)
+ - Control task time (how long it takes for the control task to run)
+
+ ![](images/timing.png)
+
+Read [this docs page](/firmware/arch/timing-manager.md) for detailed information on the timing manager
```

Comment:
I would like the formatting of the proper name of the Timing Manager to be capitalized consistently.